qc_utils/__init__.py

```diff
@@ -1,7 +1,7 @@
 from qc_utils.qcmetric import QCMetric, QCMetricRecord
 
 __title__ = "qc-utils"
-__version__ = "20.05.1"
+__version__ = "20.08.6"
```

Comment:
```suggestion
__version__ = "20.06.1"
```
I have not been clear on how I was intending to version this.. So the intention is YY.MM.X where X is the release number in that particular month, starting from 1 (i know, maybe it really should start from 0).

Reply:
Wasn't sure how to decipher the scheme, thanks